src/nifake/metadata/functions.py

```diff
@@ -1041,6 +1041,32 @@
             'note': 'Splinter is not supported.',
         },
     },
+    'reset': {
```

Comment:
I added simpleFunction function with this exact signature. We should use that for the tests - no need to have a separate reset() function.

Reply:
Using simple_function()